spec/factories/divisions.rb

```diff
@@ -22,6 +22,9 @@
   factory :division do
     description { Faker::Lorem.sentence }
     name { Faker::Company.name }
+    internal_name nil
+    parent nil
```

Comment:
`basic_projects` was just copied over from the old system, I can't recall whether defining the nil was necessary there explicitly, for legacy reasons, but since we eventually made the decision not to carry it over to this project at all, that factory will be getting deleted entirely in the slightly-more-long-term. 

Personally, I believe that between the schema definition at the top of the file and the knowledge of whether or not your tests pass without the attribute present, it's not really necessary to further annotate which fields are being excluded by explicitly assigning nil, but it's a relatively minor point.